.github/workflows/reusable-workflow-terraform.yml

```diff
@@ -66,21 +66,10 @@ jobs:
             fi
           done <<< "${{ secrets.env }}"
 
-      - name: Configure AWS credentials for profile
-        uses: aws-actions/configure-aws-credentials@v4
+      - uses: ministryofjustice/github-actions/setup-aws-profile@setup-aws-profile
         with:
-          role-to-assume: ${{ secrets.aws_profile_arn }}
-          aws-region: eu-west-2
-          role-duration-seconds: 900
-        continue-on-error: true
-
-      - name: Create AWS profile
-        run: |
-          aws configure set region ${{ env.AWS_REGION }} --profile ${{ inputs.aws_profile_name }}
-          aws configure set aws_access_key_id ${{ env.AWS_ACCESS_KEY_ID }} --profile ${{ inputs.aws_profile_name }}
-          aws configure set aws_secret_access_key ${{ env.AWS_SECRET_ACCESS_KEY }} --profile ${{ inputs.aws_profile_name }}
-          aws configure set aws_session_token ${{ env.AWS_SESSION_TOKEN }} --profile ${{ inputs.aws_profile_name }}
-        continue-on-error: true
+          role-arn: ${{ secrets.aws_profile_arn }}
+          profile-name: ${{ inputs.aws_profile_name }}
```

Comment:
I can not see this secret in https://github.com/ministryofjustice/github-actions/settings/secrets/actions

Reply:
its a reusable workflow secret, the secret is injected into the reusable workflow from the calling workflow which references the secret in GitHub Actions

Reply:
Thanks!